tests/cft/test_r2_support_bundle.py

```diff
@@ -39,6 +39,22 @@
 
 
 # pylint: disable-msg=too-many-public-methods
```

Comment:
This pylint suppression needs to be on top of class

Reply:
Done